src/script/management-sdk.ts

```diff
@@ -510,9 +509,7 @@ class AccountManager {
             }
         }
 
-        request.set("Accept", `application/vnd.code-push.v${AccountManager.API_VERSION}+json`);
-        request.set("Authorization", `Bearer ${this._accessKey}`);
-        request.set("X-CodePush-SDK-Version", packageJson.version);
```

Comment:
Do we still need `packageJson` in this file?

Reply:
Same about `API_VERSION`.

Reply:
Good catch! Thank you!